packages/bundlers/default/src/DefaultBundler.js

```diff
@@ -1036,15 +1036,21 @@ function createIdealGraph(
   }
 
   let modifiedSourceBundles = new Set();
-
   // Step Remove Shared Bundles: Remove shared bundles from bundle groups that hit the parallel request limit.
   for (let bundleGroupId of bundleGraph.getNodeIdsConnectedFrom(rootNodeId)) {
     // Find shared bundles in this bundle group.
     let bundleId = bundleGroupId;
 
     // We should include "bundle reuse" as shared bundles that may be removed but the bundle itself would have to be retained
     let bundleIdsInGroup = getBundlesForBundleGroup(bundleId); //get all bundlegrups this bundle is an ancestor of
-    if (bundleIdsInGroup.length > config.maxParallelRequests) {
+
+    // Filter out inline assests as they should not contribute to PRL
+    let numBundlesContributingToPLR = bundleIdsInGroup.reduce((count, b) => {
+      let bundle = nullthrows(bundleGraph.getNode(b));
+      return count + (bundle === 'root' || bundle.bundleBehavior !== 'inline');
```

Comment:
Yup, from my understanding `bundle` can either be a string equal to root or an object of type `Bundle`. So I need to check that the value isn't a string before attempting to access any properties of `Bundle`.

Reply:
Unfortunately, this is Flow incorrectly flagging `bundle` as possibly being `root`. `bundleIdsInGroup` should never include the root node here since we are iterating through nodes connected from the root node (`for (let bundleGroupId of bundleGraph.getNodeIdsConnectedFrom(rootNodeId)) {...}`). What you could do in cases like these is to place an invariant like so: `invariant(bundle !== 'root');`